test/custom_extension_test.dart

```diff
@@ -0,0 +1,35 @@
+library expected_output.test.custom_extension_test;
```

Comment:
Yeah that's the reason why. :frowning_face: I'll add some more content to the README regarding this shortcoming.